lambda_functions/submission_service/submit_item.py

```diff
@@ -90,3 +99,67 @@ def submit_item(event, context, is_test=False, session=None):
 
     response_cors = helper.set_cors(response, event, is_test)
     return response_cors
+
+
+def send_confirmation_mail(submission: Submission):
+    stage = os.environ['STAGE']
+    if stage == 'prod':
+        confirmation_link = 'https://api.detektivkollektiv.org/submissions/{}/confirm'.format(
+            submission.id)
+    else:
+        confirmation_link = 'https://{}.api.detektivkollektiv.org/submissions/{}/confirm'.format(
+            stage, submission.id)
+    recipient = submission.mail
+    sender = "DetektivKollektiv <info@detektivkollektiv.org>"
+    subject = 'Bestätige deine Mail-Adresse'
+    body_text = (
+        "Bitte bestätige deine Mailadresse durch Klick auf folgenden Link {}".format(confirmation_link))
+
+    body_html = """<html>
+    <head><title>Bestätige deine Mail-Adresse</title></head>
+    <body>
+    <h1 style="color: #ffcc00;">Danke fürs Einreichen deines Falles!</h1>
+    <p>Damit wir dich informieren können, sobald unsere Detektiv*innen deinen Fall gelöst haben, musst du uns noch deine E-Mail Adresse bestätigen.</p>
+    <p>Klicke dazu einfach auf folgenden Link</p>
+    <a href="{}">
+        Klicke hier, um deine Mailadresse zu bestätigen!
+    </a>
+    
+    <p>Falls du keine Inhalte beim DetektivKollektiv eingereicht hast, kannst du diese Mail einfach ignorieren.</p>
+    </body>
+    </html>
+    """.format(confirmation_link)
```

Comment:
Not sure if that's possible, but could we move the HTML part to a separate file?
This would imho greatly improve the readability.